charts/renku/requirements.lock

```diff
@@ -4,7 +4,7 @@ dependencies:
   version: 0.2.0
 - name: renku-notebooks
   repository: https://swissdatasciencecenter.github.io/helm-charts/
-  version: 0.2.0
+  version: 0.2.1-b0c93bb
```

Comment:
Fix before merging
```suggestion
  version: 0.2.1
```

Reply:
well this was actually on purpose -- because this PR was for a specific version of the notebooks service that introduced a breaking change

Reply:
Ah, I see